core/modules/entity/entity.class.inc

```diff
@@ -278,6 +319,34 @@ abstract class Entity extends stdClass implements EntityInterface {
     }
     return $values;
   }
+
+  /**
+   * Implements Backdrop\entity\EntityInterface::getRevisionId().
+   */
+  public function getRevisionId() {
+    return NULL;
+  }
+
+  /**
+   * Implements Backdrop\entity\EntityInterface::isActiveRevision().
+   */
+  public function isActiveRevision() {
+    return $this->is_active_revision;
+  }
+
+  /**
+   * Implements Backdrop\entity\EntityInterface::getActiveRevision().
+   */
+  public function getActiveRevision() {
+    return $this->active_revision;
```

Comment:
I didn't see this in the earlier iterations. I think this should be renamed to `getActiveRevisionId()` and `$this->active_revision_id` to make it clear you're getting an ID (or VID really) and not  an entire object. That would also better line up with the existing `$this->revisionId()`